public/documentation/about/faq.md

```diff
@@ -93,6 +93,18 @@ Also note that we don't have a way to package your app with Crosswalk for deploy
 
 Crosswalk does not support iOS at this time.
 
+### <a id="How-to-proguard-Crosswalk"></a>How to proguard Crosswalk?
+
```

Comment:
If I understand correctly we are saying to /not/ use ProGuard obfuscation on Crosswalk's library.  So "How to proguard Crosswalk" should become:  "How to use Crosswalk on a project using ProGuard".  Is that right?